test/cppsuite/test_harness/runtime_monitor.cxx

```diff
@@ -338,4 +351,30 @@ runtime_monitor::finish()
           "limits.");
 }
 
+/*
+ * This function generates a file that contains the values of the different statistics that need to
+ * be saved as indicated by the configuration file.
+ */
+void
+runtime_monitor::save_stats(const std::string &filename)
+{
+    std::string stat_info;
+    std::ofstream file(filename + ".json");
+    file << "[{\"info\":{\"test_name\": \"" << filename << "\"},\"metrics\": [";
+
+    for (const auto &stat : _stats) {
+        if (stat->get_save())
```

Comment:
`get_save` is a bit clunky here. 
It doesn't match the traditional getter naming pattern, but considering something like `should_save` instead.

Reply:
As discussed, I would like to keep the `get_xx` pattern for getters.